Cannot find module '.../.vscode/extensions/shengchen.vscode-checkstyle-0.5.1/client/out/src/extension'

Plugin Build: [#66](https://vscjavaci.cloudapp.net/job/vscode_checkstyle/job/vscode-checkstyle.vsix/66/Azure)
VSCode Version: 1.24.1
Platform: Win/Mac/Linux

After installed the plugin, when open a java file, will get error like "Activating extension 'shengchen.vscode-checkstyle' failed: Cannot find module 'C:\Users\v-tanwu.FAREAST\.vscode\extensions\shengchen.vscode-checkstyle-0.5.1\client\out\src\extension' ".
![image](https://user-images.githubusercontent.com/17060520/42077214-b1ee99a6-7baa-11e8-96b0-bd4d3ea54940.png)

And if you execute a checkstyle command from palatte, you will get error like "command 'checkstyle.setAutocheck' not found":
![image](https://user-images.githubusercontent.com/17060520/42077316-0dc6bcb8-7bab-11e8-9924-f958654f0183.png)

--- a/package.json
+++ b/package.json
@@ -32,7 +32,7 @@
         "onCommand:checkstyle.setPropertyFile",
         "onCommand:checkstyle.setAutocheck"
     ],
-    "main": "./client/out/src/extension",
+    "main": "./client/out/extension",
     "contributes": {
         "commands": [
             {

